### ISSUE TITLE
Can we add text in links ?

### ISSUE THREAD

**Issue by shaddu** [jpb12/react-tree-graph#48]
Is there any way to add text for links connecting two nodes ?

**Comment by jpb12** [comment]
This is not currently possible.

**Comment by shaddu** [comment]
@jpb12 will love to give it a try if u can add some steps on how to run it locally

**Comment by TomLisankie** [comment]
Just curious, has this been addressed and just not closed?

**Comment by jpb12** [comment]
No, this has not been addressed. It would help if someone interested in this could go into more details about how they want this to behave, or what their use case is

One way I thought of implementing this was [textpath](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/textPath), but I'm not sure if people would want the text curving with the path like this, or instead would prefer a more complex solution positioning a `text` node.
